core/src/main/java/org/hisp/dhis/android/core/common/FeatureType.kt

```diff
@@ -25,39 +25,28 @@
  *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  *  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+package org.hisp.dhis.android.core.common
 
-package org.hisp.dhis.android.core.common;
-
-public enum FeatureType {
+import kotlinx.serialization.Serializable
 
+@Serializable
```

Comment:
There is a Serializable annotation in the `core` package, but I guess this is unavoidable if we don't want to duplicate the enums